noise_delta_t: 1.0
map_frame_id: 'map'
odom_frame_id: 'zed_objdet_odom'
field_dims:
  x_min: -6.0
  x_max: 6.0
  y_min: -6.0
  y_max: 6.0

init_dims:
  x_min: -6.0
  x_max: 6.0
  y_min: -6.0
  y_max: 6.0

num_particles: 200

noise_stdev:
  position: 0.025
  rotation: 0.005

  # Modifiers for noise stddev when robot is moving or stopped
  # Allows for more accurate localization when stopped by using
  # a lower stddev for noise when the robot is at rest.
  stopped_pos_noise_multiplier: 1.0
  stopped_rot_noise_multiplier: 1.0
  moving_pos_noise_multiplier: 1.0
  moving_rot_noise_multiplier: 1.0

rotation_threshold: 0.25

camera_sigmas: [ 1.000, 1.000 ]

bearing_only: false

beacons:
  - [0.0, 1.0, "8"]
  - [0.0, 1.91, "2"]